types/google.visualization/index.d.ts

```diff
@@ -72,10 +72,19 @@ declare namespace google {
         //#endregion
         
         //#region data
+        // https://developers.google.com/chart/interactive/docs/reference#google_visualization_data_group
+        export interface GroupColumnOptions {
+            column: number;
+            aggregation: (values: any[]) => any;
+            type: string;
+            label?: string;
+            id?: any;
+        }
+        
         export class data {
             // https://developers.google.com/chart/interactive/docs/reference#group
-            static group(data: DataTable, keys: any[], columns: any[]): DataTable;
-            static group(data: DataView, keys: any[], columns: any[]): DataTable;
+            static group(data: DataTable, keys: any[], columns?: GroupColumnOptions[]): DataTable;
```

Comment:
Please create a separate interface for `keys` also.
And use that like this
`(number|newTyle)[]`

Reply:
Done, added modifier func and re-ordered the class to make more sense with the document citations